main.go

```diff
@@ -67,6 +67,7 @@ func help(w http.ResponseWriter, r *http.Request) {
 
 func main() {
 	flag.IntVar(&flagPort, "port", 9340, "Port to listen for HTTP requests.")
+	flag.BoolVar(&flagVolume, "quote.volume", false, "Exports volume.")
```

Comment:
Is there a reason why this feature must be hidden behind a flag? If it's not an expensive feature, you could turn it on by default and then there's no need for an extra command line flag.

Reply:
The reason was to suppress command line flag changes.
But, I understand that it was unnecessary thing as you say. I'll remove the flag.